lib/swoosh/email/smtp.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Swoosh.Email.SMTP do
```

Comment:
I'd prefer if this module was closer to the adapters, as it's only used there. Annoyingly I can't really think of a good name for it.